services/static-webserver/client/source/resource/osparc/tours/tis_tours.json

```diff
@@ -98,46 +98,46 @@
     "context": "osparc-test-id=creditsIndicatorButton",
     "steps": [{
       "title": "Credits",
-      "text": "In order to run Personalized TI Plans, credits are required.<br>In the next Guided Tour you will learn how to purchase them."
+      "text": "In order to run personalized TI plans, Credits are required. In the next steps you will learn how to purchase them."
     }, {
       "beforeClick": {
         "selector": "osparc-test-id=creditsIndicatorButton",
         "event": "tap"
       },
       "anchorEl": "osparc-test-id=creditsNavBarContainer",
-      "text": "By clicking on the Credits Indicator, you will you see the available Credits. Tap the three dots to access the Billing Center.",
+      "title": "Credits Indicator",
+      "text": "By clicking on the Credits indicator, you will you see the number of available Credits. Click the three dots to access the Billing Center.",
       "placement": "left"
     }, {
       "beforeClick": {
         "selector": "osparc-test-id=billingCenterButton",
         "action": "execute"
       },
       "anchorEl": "osparc-test-id=walletsList",
-      "title": "Credits Accounts",
-      "text": "This is the list of Credits Accounts you have access to. You have your own Account, but you could also use Credit Accounts that were shared with you.",
+      "title": "Credit Accounts",
+      "text": "This lists the Credit Accounts you have access to. Apart from your own Account, the list also includes Credit Accounts that were shared with you.",
       "placement": "bottom"
     }, {
       "anchorEl": "osparc-test-id=personalWalletsList",
-      "title": "Top-up Credits",
-      "text": "There are two ways to charge credits into your Credit Account: you can set up an Auto-recharge process or go for a One time payment.",
+      "title": "Buy Credits",
+      "text": "There are two ways to add Credits to your Credit Account: you can set up an auto-top-up procedure or make a one-time payment.",
```

Comment:
_There are two ways to add Credits to your Credit Account: you can set up an auto-top-up procedure or make a one-time payment._

**There are two ways to add Credits to your account: you can set up and select the auto-top-up option or by making one-time payments.**

Reply:
https://github.com/ITISFoundation/osparc-simcore/pull/6112#issuecomment-2252735810